tf-modules/cumulus-test-cleanup/reap-kinesis.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+// handy script to delete any old test kinesis streams that are created but not
+// cleaned up.
+
+const aws = require('aws-sdk');
+const moment = require('moment');
+const kinesis = new aws.Kinesis();
+
+const deleteOlderThanDays = 1;
+
+const getStreams = async () => {
+  return kinesis.listStreams({})
+    .promise()
+    .then((result) => result.StreamNames);
+};
+
+
+const filterOld = (streams) => {
+  const matcher = /(Error|Trigger|SourceTest)-([0-9]{13})-(Kinesis|Lambda)/;
+  const results = streams.map((s) => {
+    if (s.match(matcher)) {
+      const streamDate = Number(s.match(matcher)[2]);
+      if (moment(new Date()).diff(streamDate, 'days') > deleteOlderThanDays) {
```

Comment:
What's the functional difference between 
moment() 

and 

moment(new Date())?

Reply:
As far as I can tell, nothing, so I updated it.